content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -205,7 +205,7 @@ spec:
     - conditionType: "www.example.com/feature-1"
 status:
   conditions:
-    - type: Ready                              # a built in PodCondition
+    - type: Ready                              # a builtin PodCondition
```

Comment:
```suggestion
    - type: Ready                              # a built in PodCondition
```

”built in” as two words is more idiomatic when built is used as a verb. The one word form is using “builtin” as a noun.